internal/app/enaptercli/cmd_rule_engine_rule_logsf.go

```diff
@@ -0,0 +1,70 @@
+package enaptercli
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+type cmdRuleEngineRuleLogsf struct {
+	cmdRuleEngineRule
+	ruleID string
+}
+
+func buildCmdRuleEngineRuleLogsf() *cli.Command {
+	cmd := &cmdRuleEngineRuleLogsf{}
+	return &cli.Command{
+		Name:               "logsf",
+		Usage:              "Stream rule logs",
+		CustomHelpTemplate: cmd.HelpTemplate(),
+		Flags:              cmd.Flags(),
+		Before:             cmd.Before,
+		Action: func(cliCtx *cli.Context) error {
+			return cmd.do(cliCtx)
+		},
+	}
+}
+
+func (c *cmdRuleEngineRuleLogsf) Flags() []cli.Flag {
+	return append(c.cmdRuleEngineRule.Flags(),
+		&cli.StringFlag{
+			Name:        "rule-id",
+			Usage:       "Rule ID or slug to update",
```

Comment:
Usage description is invalid. Should be something like `ID or slug of a rule to stream logs of`.